notebooks/community/vision/image_warehouse_sdk.ipynb

```diff
@@ -0,0 +1,1062 @@
+{
```

Comment:
<p>combine the 2 install steps</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Done</p>